lib/method.js

```diff
@@ -75,6 +75,8 @@ const identity = function identity(x) {
 module.exports = function(spec) {
   const requestMethod = (spec.method || 'GET').toUpperCase();
   const encode = spec.encode || identity;
+  const json = spec.json !== false;
+  const contentType = spec.contentType;
```

Comment:
Do you mind also setting the default value here (and updating the values in the test), like you have have for `json`?